components/Banners/docs/README.md

```diff
@@ -0,0 +1,5 @@
+# Banners
```

Comment:
I think I understand why you renamed this to Banners (because material.io refers to it in the plural form). However, we only expect to ship one Banner implementation in this directory so the previous name was technically correct. Similarly, AppBar is not termed AppBars.

Reply:
Sure, I was confused by the existing `Banner` directory there.  Renamed it back to `Banner` in 13e12dd.